<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgExportStoryboard</class>
 <widget class="QWidget" name="WdgExportStoryboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Storyboard</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Frame number of first item to render:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Frame number of last item to render:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Rows per page :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Columns per page :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Font Size :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Page Size :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Page Orientation :</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QSpinBox" name="spinboxFirstItem"/>
         </item>
         <item>
          <widget class="QSpinBox" name="spinboxLastItem"/>
         </item>
         <item>
          <widget class="QSpinBox" name="spinboxRow"/>
         </item>
         <item>
          <widget class="QSpinBox" name="spinboxColumn">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinboxFontSize"/>
         </item>
         <item>
          <widget class="QComboBox" name="cmbPageSize">
           <item>
            <property name="text">
             <string>A4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A5</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbPageOrient">
           <item>
            <property name="text">
             <string>Portrait</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Landscape</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btnSpecifySVG">
       <property name="text">
        <string>Speciify layout using SVG file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblSvgFileName">
       <property name="text">
        <string>svgFileName</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
